README.md

```diff
@@ -145,6 +145,12 @@
   - [philippe-vandermoere/terraform-provider-algolia](https://github.com/philippe-vandermoere/terraform-provider-algolia)
   - [brightnetwork/terraform-provider-algolia](https://github.com/brightnetwork/terraform-provider-algolia)
   - [SectorLabs/terraform-provider-algolia](https://github.com/SectorLabs/terraform-provider-algolia)
+- Flutter:
```

Comment:
Those libraries are not community libraries, but official ones, the following can be added instead:
- _API Client_
  - Dart -  https://github.com/algolia/algoliasearch-client-dart
- _Libraries & Tools_
  - Flutter Helper - https://github.com/algolia/algoliasearch-helper-flutter